### PR TITLE
WCAG exemptions in regulatory work

### PR DESCRIPTION
**Pull request by maryjom**
This PR is superceded by #367 to avoid merge conflicts and other messiness since this was open so long. Closing this PR without merging.

### CODE REVIEW

**Review by bruce-usab — COMMENTED** [review]
In survey, I suggested stronger phrasing.

**Comment by mraccess77** [comment]
I think it's worth talking about what US DOJ said in latest advance copy of final rule
https://www.ada.gov/assets/pdfs/web-rule.pdf

In determining how to make conventional electronic documents conform to WCAG 2.1
Level AA, public entities may find it helpful to consult W3C’s guidance on non-web information
and communications technology, which explains how the WCAG success criteria can be applied
to conventional electronic documents. The Department believes the compliance dates discussed
in § 35.200(b) will provide public entities sufficient time to understand how WCAG 2.1
Level AA applies to their conventional electronic documents.

File: introduction.md [hunk 1]
```diff
@@ -31,10 +31,16 @@ While WCAG 2 was designed to be technology-neutral, it assumes the presence of a
...
-<div class="note">Sets of software that meet this definition appear to be extremely rare.</div>
+<div class="note">
+  
+Sets of software that meet this definition appear to be extremely rare.</div>
+
+Not all success criteria have been fully adopted in all local regulations and legislation, and may not be applicable to all technologies. WCAG2ICT was also used to determine whether or not to apply certain success criteria. For example, some local standards such as Section 508 in the US, and EN 301 549 in Europe, state that non-Web documents and non-Web software do not need to comply with WCAG 2.0 Success Criteria 2.4.1 Bypass Blocks, 2.4.5 Multiple Ways, 3.2.3 Consistent Navigation, and 3.2.4 Consistent Identification. In addition, EN 301 549 also states that non-Web software does not need to comply with 2.4.2 Page titled and 3.1.2 Language of parts. Regulators should consider the applicability of individual success criteria to non-web documents and software.
```
Comment (mitchellevan) [inline introduction.md:43]:
```suggestion
Not all success criteria have been fully adopted in all local regulations and legislation, and may not be applicable to all technologies. Regulators have used WCAG2ICT to determine whether or not to apply certain success criteria. For example, some local standards such as Revised 508 Standards (2017) in the US, and EN 301 549 version 3.2.1 (2021) in Europe, have stated that non-Web documents and non-Web software do not need to comply with WCAG 2.0 Success Criteria 2.4.1 Bypass Blocks, 2.4.5 Multiple Ways, 3.2.3 Consistent Navigation, and 3.2.4 Consistent Identification. In addition, EN 301 549 has also stated that non-Web software does not need to comply with 2.4.2 Page Titled and 3.1.2 Language of Parts. Regulators should consider the applicability of individual success criteria to non-web documents and software, and where necessary consider alternate accessibility requirements to address user needs.
```

I agree with @GreggVan's [concern](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-survey-comment-answers/results) that the references could become dated. To address this concern, as well as for writing style, I propose changing most of the paragraph to past tense. I also propose citing specific versions of the regulations, so our statements here will remain true regardless of future versions of those regulations.

Finally, I added a bit about "address user needs" at the end.

Comment (bruce-usab) [reply introduction.md:43]: +1 to @mitchellevan edits

**Review by ChrisLoiselle — APPROVED** [review]

**Pull request closed without merging**